plugin-gestures/src/main/java/com/mapbox/maps/plugin/gestures/GesturesPluginImpl.kt

```diff
@@ -675,6 +688,7 @@ class GesturesPluginImpl : GesturesPlugin, GesturesSettingsBase, MapStyleObserve
 
   internal fun handleScaleBegin(detector: StandardScaleGestureDetector): Boolean {
     quickZoom = detector.pointersCount == 1
+    deferredRotate = 0.0
```

Comment:
applies for several similar code blocks below

Reply:
Well it will conflict with the current `end` logic - since there a different animation is applied based on the velocity and it's not really clear how to mix those two. I'd just allowed the last value to be silently ignored, that's not really noticeable.